pandas/tests/groupby/aggregate/test_aggregate.py

```diff
@@ -331,6 +331,37 @@ def test_uint64_type_handling(dtype, how):
 
 class TestNamedAggregation:
```

Comment:
could make this a separate class

Reply:
You're saying split the series specific ones, into TestNamedAggregationSeries and TestNamedAggregationDataFrame?

Reply:
yes